### ISSUE TITLE
Accent marks not working properly

### ISSUE THREAD

**Issue by nicomagalhaes** [debauchee/barrier#1160]
I'm using Barrier to integrate a PC and a laptop (PC as server), and several times the accent marks do not work properly. Sometimes, they do not appear, and sometimes they appear twice.

1. Try to write something with accent marks
2. Results on the Server:
nao está funcionando
nao está funcionando
não esta funcionando
não esta funcionando
não está funcionando
3. Results on the Client:
nãõ est´a funcionando
nãõ est´á funcionando
nao est´á funcionando
nãõ est´á funcionando
nao est´á funcionando

The sentence should be the following on both Server and Client:
não está funcionando

**Desktop (please complete the following information):**

 - OS: Windows 10
- Barrier 2.3.3

**Additional context**

I'm using the keyboard on the server connected by a USB cable.

**Comment by janevert** [comment]
Same issue here, using barrier server on Ubuntu (21.04) and client on Win10. Both barrier are version 2.3.3.
For me, the affected letters are ' " ~ (single quote, double quote and tilde).

**Comment by LaraArgento** [comment]
same here. I have to Reload Barrier in order to fix the issue, but it comes back in a matter of a few minutes and it is very annoying.

**Comment by stephen-neumeyer** [comment]
tilde (~) does not work across barrier (on client system).  server: mac osx 'monterey' client: ubuntu 20.04.3 LTS.  i turned off SSL on client & server.  i'm using mostly default options, on the client side, i manually entered the server's ip address.  (i'm inside a subnet for both systems).

**Comment by abastola0** [comment]
You can Map the ~ and ` to themselves again in the hotkeys section. Worked for me !!

**Comment by bobstro** [comment]
> You can Map the ~ and ` to themselves again in the hotkeys section. Worked for me !!

Mapping in server settings will send the correct keys to the client, but if you press them with focus on the server, the keys are sent to the client and not usable on the server.

**Comment by felipe-vf** [comment]
> You can Map the ~ and ` to themselves again in the hotkeys section. Worked for me !!

Could you provide a screenshot of how the configuration looks like? I tried mapping the hotkeys but the problem still persists on the client side. Both computers are running Windows.

**Comment by abastola0** [comment]
Create a hotkey and select the action for the same. In this case tilde or any other keys that don't work on the client. Remember this way works only for the client. For example: if you set tilde this way, the tilde would work on the client but on on the server. See image attached !
![barrierTest](https://user-images.githubusercontent.com/41137160/178304103-9d5a4b2e-2991-4132-8b51-27dc0f4a11da.jpg)

**Comment by abastola0** [comment]
> > You can Map the ~ and ` to themselves again in the hotkeys section. Worked for me !!
> 
> Mapping in server settings will send the correct keys to the client, but if you press them with focus on the server, the keys are sent to the client and not usable on the server.

yeah is there any possible solution to it apart from setting up the hotkey?
